core/tests/test_util_jobs.py

```diff
@@ -124,18 +163,31 @@ class DeleteExpCopiesMRJobManager(
     """
     @classmethod
     def _get_continuous_computation_class(cls):
+        """Returns the DeleteExpCopiesAggregator class associated with this
+        MapReduce job.
+        """
         return DeleteExpCopiesAggregator
 
     @classmethod
     def entity_classes_to_map_over(cls):
+        """Returns the ExplorationModel domain object."""
         return [exp_models.ExplorationModel]
 
     @staticmethod
     def map(item):
+        """Deletes the exploration with the given id if it is of the 'Copies'
+        category.
+
+        Args:
+            item: An exploration domain object.
+        """
         if item.category == 'Copies':
             exp_services.delete_exploration(
                 feconf.SYSTEM_COMMITTER_ID, item.id, force_deletion=True)
 
     @staticmethod
     def reduce(exp_id, list_of_exps):
+        """Deletes the saved and published copies of the existing exploration
+        to be implemented.
+        """
```

Comment:
The deletion is done in the map function above itself.

Reply:
Sorry, I am not sure what this method is actually for as it is not implemented yet.